### PR TITLE
Dj 2.2

### PR DESCRIPTION
**Pull request by madhulika95b**
@ldko @somexpert 
This PR upgrades to Django 2.2

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1 +1 @@
-Django~=1.11.21
+Django~=2.2
```
[the remark this answers is not in the document]
Comment (madhulika95b) [reply requirements.txt:1]: ok, I'll update it to the latest patch release `2.2.17`.

**Review by somexpert — APPROVED** [review]
LGTM! :thumbsup:

**Review by ldko — COMMENTED** [review]
Hi @madhulika95b, The app is working for me in Django 2.2! A couple small things:
1. You should add your name to the contributors on the README.
2.  We should update the [quickstart doc](https://github.com/unt-libraries/django-invite/blob/68d9e47e7590206749358c58dc2dc573a005904b/docs/quickstart.rst) to reference the new version of django-invite and maybe change the urls.py snippet to use `path`.

**Review by ldko — APPROVED** [review]
Thank you, Madhulika!

**Pull request merged**